odoo/modules/module.py

```diff
@@ -138,9 +134,10 @@ def initialize_sys_path():
     sys.modules["odoo.addons.base.maintenance.migrations"] = upgrade
 
     # hook deprecated module alias from openerp to odoo and "crm"-like to odoo.addons
-    if getattr(initialize_sys_path, 'called', False): # only initialize once
+    if not getattr(initialize_sys_path, 'called', False): # only initialize once
         sys.meta_path.insert(0, OdooHook())
         sys.meta_path.insert(0, AddonsHook())
+        initialize_sys_path.called = True
```

Comment:
Even if it doesn't change the logic, this line was unindented in 13.0.

As a thumb rule, if you can keep the same code as in 13.0, it will avoid conflicts when forward-porting future patches about the multi call of this method.

Reply:
@odony has amended my changes in 13.0 just before r+, and indented this line in the if block:
>  @odony odony [force-pushed](https://github.com/odoo/odoo/compare/0b89d4947555203fc996b071cdbb685364650cfb..af52ab19b2430036047a20c2ae2c00b925f61e26) the odoo-dev:13.0-fix_meta_path_bloating-pmo branch from 0b89d49 to af52ab1 9 hours ago